pkg/manager/manager_rules.go

```
@@ -316,3 +531,189 @@ func SyncLocalRules(cursorDir string) error {
+	for _, tmpl := range allTemplates {
+		// Check if the template name matches our pattern
+		if !matchGlob(g, tmpl.Name) {
+			continue
+		}
+
+		// Try to add this template directly instead of calling AddRule
+		content := tmpl.Content
+		targetPath := filepath.Join(cursorDir, tmpl.Name+".mdc")
+
+		// Write to .cursor/rules/{ruleName}.mdc
+		err := os.WriteFile(targetPath, []byte(content), 0o644)
```

golangci-lint (1.64.8)

[medium] G306: Expect WriteFile permissions to be 0600 or less

(gosec)